Svc/ComLogger/test/ut/ComLoggerTester.cpp

```diff
@@ -185,7 +185,7 @@ namespace Svc {
         }
 
         // Make sure we reached the end of the file:
-        FwSignedSizeType length = static_cast<FwSignedSizeType>(sizeof(FwSignedSizeType));
+        FwSizeType length = static_cast<FwSizeType>(sizeof(FwSizeType));
```

Comment:
Technically this should say "sizeof(U16)" as that is the size type of the com logger